docs/sql/binary.md

```diff
@@ -45,4 +45,29 @@ aggregation functions `BIT_AND`, `BIT_OR`, and `BIT_XOR`.
     <td>Generate a `VARCHAR` string describing the value in hexadecimal</td>
     <td><code>TO_HEX(x'0abc')</code> => <code>'0ABC'</code></td>
   </tr>
+  <tr>
+    <td><code>POSITION</code>(binary1 IN binary2)</td>
+    <td>Returns the position of the first occurrence of binary1 in binary2</td>
```

Comment:
I suspect the position is in bytes.
If the first offset is 1 you need to document that.

Reply:
you have to say what the return value is if the string is not found